.github/workflows/publish.yml

```diff
@@ -1,28 +1,33 @@
-name: Publish on tag
+name: Publish
 
 on:
   push:
     tags:
-      - '*'
+      - '*v*'
+
+env:
+  FORCE_COLOR: 3
+  PIP_DISABLE_PIP_VERSION_CHECK: 1
 
 jobs:
   publish:
+    name: Publish
     runs-on: ubuntu-latest
     steps:
```

Comment:
FYI: I am intending to revert some of these changes in my next misc PR. Most of our YAML files look consistent, in terms of `@v3`, having `env` at the bottom of the file, the order of the steps, etc, and this change breaks some of this inconsistency.

In the future, if you make these changes, then at least try to be globally consistent/uniform.

Reply:
Could you elaborate why env should be at the bottom of the file?

Reply:
It doesn't. My argument is in favor of consistency. `env` could equally be in the beginning, but if so, let's keep all files the same way.

Reply:
I find the env at the top to be more readable as for other changes no preference :)